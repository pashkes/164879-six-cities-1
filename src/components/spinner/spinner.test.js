import React from "react";
import {shallow, configure} from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import Spinner from './spinner.jsx';
configure({adapter: new Adapter()});

it(`Spinner renders correctly`, () => {
  const signIn = shallow(<Spinner />);
  expect(signIn).toMatchSnapshot();
});
