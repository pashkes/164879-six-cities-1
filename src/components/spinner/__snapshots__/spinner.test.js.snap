// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner renders correctly 1`] = `
<div
  aria-label="loading..."
  className="spinner spinner-1"
/>
`;
